blobstore/src/test/java/org/jclouds/blobstore/integration/internal/BaseContainerIntegrationTest.java

```diff
@@ -553,7 +553,7 @@ protected void addTenObjectsUnderPrefix(String containerName, String prefix) thr
 
    protected void awaitConsistency() {
       if (view.getConsistencyModel() == ConsistencyModel.EVENTUAL) {
-         Uninterruptibles.sleepUninterruptibly(10, TimeUnit.SECONDS);
+         Uninterruptibles.sleepUninterruptibly(20, TimeUnit.SECONDS);
```

Comment:
Can you just override this in `SwiftContainerIntegrationLiveTest` if needed?  This sleep is really unfortunate since it slows down all tests unnecessarily, not just the uncommon case of eventual consistency.